server/gokbg3/grails-app/domain/org/gokb/cred/TitleInstancePackagePlatform.groovy

```diff
@@ -217,11 +222,14 @@ class TitleInstancePackagePlatform extends KBComponent {
     firstEditor column: 'tipp_first_editor', type: 'text'
     parentPublicationTitleId column: 'tipp_parent_publication_id'
     precedingPublicationTitleId column: 'tipp_preceding_publication_id'
+    supercedingPublicationTitleId column: 'tipp_superceding_publication_title_id'
```

Comment:
Superseding is written with 's', cf. https://dict.leo.org/englisch-deutsch/supersede